frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -95,6 +97,14 @@ export default class Entity extends React.Component<Props> {
         this.props.selectEntity(this.props.entity);
     };
 
+    getSiblingEntities = (e: SyntheticMouseEvent<HTMLLIElement>) => {
+        if (e.currentTarget && e.currentTarget.classList.contains('status')) {
+            return null;
+        }
```

Comment:
We don't need this block here: it's used in `selectEntity()` to (poorly, see point 2 below) prevent entity selection if user clicks on the `status` icon (which toggles batch editing). So you can simply remove it.

However, there are two things we should do:

1. As you'll notice, when you click on the TEST button on the entity that's not currently selected for editing, it gets selected. We want to prevent that. There's a [`stopPropagation`](https://developer.mozilla.org/en-US/docs/Web/API/Event/stopPropagation) on the event object that helps with that.
2. Unrelated to this patch (but since we're already at it, let's do it now) you should also call `e.stopPropagation()` early in the `toggleForBatchEditing()` function. Then you can remove the `if (e.currentTarget)` block from `selectEntity`, including the weird flow comment. That's not only more proper and elegant solution to the problem - it also works better. Right now, if you're logged out (i.e. unabled to do batch editing) and click on the status icon, the entity also doesn't get selected, but it should.